web-client/src/business/useCases/externalDocument/generateExternalDocumentTitle.ts

```diff
@@ -14,6 +15,11 @@ export const generateExternalDocumentTitle = (
   applicationContext: ClientApplicationContext,
   { documentMetadata },
 ) => {
+  const entry = Object.values(SYSTEM_GENERATED_DOCUMENT_TYPES).find(
+    entry => entry.eventCode === documentMetadata.eventCode,
+  );
+  if (entry) return entry.documentTitle;
```

Comment:
👍 This seems better. I would do this and get rid of the changes to `web-api/src/business/useCases/docketEntry/updateDocketEntryMetaInteractor.ts`